Snabble/Core/API/SnabbleAPI.swift

```diff
@@ -303,16 +303,30 @@ public enum SnabbleAPI {
 
 /// SnabbleSDK application user identification
 public struct AppUserId {
-    /// the public part of the `userId`
+    /// the actual information of the `userId`
```

Comment:
Klingt für mich wie ein Germanismus - warum nicht einfach "the `userId`"?

Reply:
Wegen des Klassennamen `AppUserId` 🤔
Vielleicht so? 50c9ed388fc953a79a8adb4e0141f6d5e5342cb3

Reply:
Habe nochmal Anpassungen vorgenommen, aber würde es bei `value` belassen, da es für den externen Entwickler einfacher zu verstehen ist durch `appUserId.value` anstatt `appUserId.userId`.

Reply:
9cbdc615119b2c85dcc255d3a12f2df2a56235ca